### ISSUE TITLE
Loss

### ISSUE THREAD

**Issue by myla01** [starreeze/efuf#6]
Thanks for your great work!
The paper mentions that the loss consists of three components, but the code introduces an additional `loss_gold_weighted`.
`loss = loss_pos_weighted + loss_gold_weighted + loss_sent_weighted - loss_neg_weighted`

**Comment by starreeze** [comment]
Sorry for the late reply.

`loss_gold` is an external surpervising signal from the training dataset of the original model. In our case, it is the `mixed_665k` from the llava-1.5 training dataset.
We later found it could further prevent catastrophic forgetting and yield a better performance.

**Comment by myla01** [comment]
Is this loss component not presented in the paper?

**Comment by starreeze** [comment]
No. But it's as simple as a minor trick.
